api/api.proto

```diff
@@ -1793,6 +1793,14 @@ service OpenStorageVolume {
         body: "*"
       };
     }
+
+    rpc VolumeCatalog(SdkVolumeCatalogRequest)
```

Comment:
This needs a comment as if it was explained to a non-portworx developer. Maybe add an example also.